src/commands/registries/azure/deployImageToAzure.ts

```diff
@@ -55,11 +60,64 @@ export async function deployImageToAzure(context: IActionContext, node?: RemoteT
 
     const site = nonNullProp(wizardContext, 'site');
     const createdNewWebApp: string = `Successfully created web app "${site.name}": https://${site.defaultHostName}`;
+    if (node instanceof AzureTagTreeItem) {
+        let siteClient = new SiteClient(site, node.parent.parent.parent.root);
+        let appUri: string = (await siteClient.getWebAppPublishCredential()).scmUri;
+        await createWebhookForApp(node, wizardContext, appUri);
+    } else if (node instanceof DockerHubTagTreeItem) {
+        // point to dockerhub to create a webhook
+        // http://cloud.docker.com/repository/docker/<registryName>/<repoName>/webHooks
+        const dockerhubPrompt: string = "Copy web app endpoint and browse to dockerhub";
+        let response: string = await vscode.window.showInformationMessage("Please browse to your dockerhub account to set up a CI/CD webhook", dockerhubPrompt);
+        if (response) {
+            let appUri = `https://${site.name}.scm.azurewebsites.net/docker/hook`;
+            // tslint:disable-next-line:no-unsafe-any
+            await vscode.env.clipboard.writeText(appUri);
+            // account.name, repository.name
+            // tslint:disable-next-line:no-unsafe-any
+            vscode.env.openExternal(vscode.Uri.parse(`https://cloud.docker.com/repository/docker/${node.parent.parent.parent.username}/${node.parent.repoName}/webHooks`));
```

Comment:
Use this: https://github.com/microsoft/vscode-docker/blob/master/src/utils/openExternal.ts